app/packages/app/src/components/ImageContainerHeader.tsx

```diff
@@ -54,35 +60,32 @@ const SliderContainer = styled.div`
   padding-right: 1rem;
 `;
 
-const Count = () => {
-  const element = useRecoilValue(fos.elementNames);
-  const total = useRecoilValue(
-    fos.count({ path: "", extended: false, modal: false })
-  );
-
-  return (
-    <RightDiv>
-      <div>
-        <PathEntryCounts modal={false} path={""} />
-        &nbsp;
-        {total === 1 ? element.singular : element.plural}
-      </div>
-    </RightDiv>
-  );
-};
-
 const ImageContainerHeader = () => {
+  const type = useRecoilValue(fos.dataset).mediaType;
   const setGridZoom = useSetRecoilState(gridZoom);
   const gridZoomRangeValue = useRecoilValue(gridZoomRange);
   const theme = useTheme();
   const group = useRecoilValue(isGroup);
+  const element = useRecoilValue(fos.elementNames);
+  const total = useRecoilValue(
+    fos.count({ path: "", extended: false, modal: false })
+  );
```

Comment:
This is going to suspend, which will yield a bad loading state. It should be used in a component within a `Suspense` like `Count` was previously.